examples/audio/audio_control.rs

```diff
@@ -40,9 +40,9 @@ fn pause(keyboard_input: Res<Input<KeyCode>>, music_controller: Query<&AudioSink
 fn volume(keyboard_input: Res<Input<KeyCode>>, music_controller: Query<&AudioSink, With<MyMusic>>) {
     if let Ok(sink) = music_controller.get_single() {
         if keyboard_input.just_pressed(KeyCode::Plus) {
-            sink.set_volume(sink.volume() + 0.1);
+            sink.set_volume(VolumeLevel::Amplitude(sink.volume().amplitude() + 0.1));
```

Comment:
Feels like we should implement some of the math traits for VolumeLevel and f32, so users don't have to convert types. But maybe not might get awkward since users won't always know what enum variant they're adding to.

This increased verbosity isn't great though for users wanting to do math on volume levels. Does kira have this issue?

Reply:
Kira doesn't implement any math operations for its [`Volume`](https://docs.rs/kira/latest/kira/enum.Volume.html) type. Personally, I think that makes sense, as the two variants are not linearly related. If this was a simple case of (for example) Feet and Metres, then the math operations would be consistent. Scaling a volume by 1.3 does completely different things depending on whether you are in decibels or amplitude, because decibels is a logarithmic scale, and amplitude is a linear one.